rest/oidc_api.go

```diff
@@ -256,3 +311,19 @@ func (h *handler) getOIDCCallbackURL(providerName string, isDefault bool) string
 	}
 	return callbackURL
 }
+
+// setStateCookie sets the state cookie.
+func setStateCookie(res http.ResponseWriter, value string) {
```

Comment:
Typically `w` is used for `http.ResponseWriter`, to prevent confusion between `resp *http.Response` and `req *http.Request`

The http package has good examples of variable naming:
E.g: https://golang.org/pkg/net/http/#Error

```suggestion
func setStateCookie(w http.ResponseWriter, value string) {
```

Reply:
Added as reference for `http.ResponseWriter`, to prevent confusion between resp `*http.Response` and `req` `*http.Request`